chemreg/compound/tests/test_views.py

```diff
@@ -36,3 +36,35 @@ def test_definedcompound_override():
     assert err.value.status_code == 400
     assert err.value.default_code == "invalid"
     assert err.value.args[0] == "invalid query parameter: overRide"
+
+
+@pytest.mark.django_db
+def test_definedcompound_detail_attrs(user_factory, defined_compound_factory):
+    """Test that passing an override query parameter works in DefinedCompoundViewSet."""
```

Comment:
Looks like this is a docstring for another test.

Reply:
good catch. This could be proof of aliens jacking my keybindings again. I'll have to look into it further.